Error: Cannot find module 'ajv/dist/compile/codegen'

"fork-ts-checker-webpack-plugin": "^7.0.0"
"typescript": "^4.5.5",
"webpack": "^5.68.0",
"webpack-cli": "^4.9.2",

this morning i deleted my node-modules and package-lock.json to have a clean install.
Unfortunately i hit an error that traces back to this plugin as a start

can you help to determine what can be done to solve this ?

```
[webpack-cli] Error: Cannot find module 'ajv/dist/compile/codegen'
Require stack:
- /node_modules/ajv-keywords/dist/definitions/typeof.js
- /node_modules/ajv-keywords/dist/keywords/typeof.js
- /node_modules/ajv-keywords/dist/keywords/index.js
- /node_modules/ajv-keywords/dist/index.js
- /node_modules/schema-utils/dist/validate.js
- /node_modules/schema-utils/dist/index.js
- /node_modules/fork-ts-checker-webpack-plugin/lib/plugin.js
- /node_modules/fork-ts-checker-webpack-plugin/lib/index.js
```

thanks

--- a/package.json
+++ b/package.json
@@ -63,7 +63,7 @@
     "fs-extra": "^10.0.0",
     "memfs": "^3.4.1",
     "minimatch": "^3.0.4",
-    "schema-utils": "4.0.0",
+    "schema-utils": "^3.1.1",
     "semver": "^7.3.5",
     "tapable": "^2.2.1"
   },

